### DISCUSSION TITLE
Flying units behaviour like in H3

### DISCUSSION THREAD

**Discussion by Zaratul555** [ihhub/fheroes2#5085] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Suggestion for option for flying units behave like in H3. I mean no flying through all the battlefield but only move as far as move points tell.

### Describe the possible solution

Add this feature in the gameplay options please.

**Comment by ihhub** [comment]
I am converting this issue into a discussion topic as it is definitely outside the project's goal.
